exercises/practice/resistor-color/tests/resistor-color.rs

```diff
@@ -0,0 +1,64 @@
+use resistor_color::ResistorColor;
```

Comment:
```suggestion
use resistor_color::{colors, color_to_value, value_to_color_string, ResistorColor};
```

Just makes usage a bit cleaner to import the functions as well.